build.gradle

```diff
@@ -10,8 +10,9 @@ plugins {
   id 'org.checkerframework' version '0.5.16' apply false
   id 'org.jetbrains.grammarkit' version '2020.3.2' apply false
   id 'org.jetbrains.intellij' version '0.7.2' apply false
-  // Minor Kotlin version is effectively enforced by the IntelliJ version we're building against
-  id 'org.jetbrains.kotlin.jvm' version '1.3.72' apply false
+  // Minor Kotlin version is effectively enforced by the IntelliJ version we're building against;
+  // e.g. 2020.x uses version 1.3.x amd 2021.1 uses 1.4.x
+  id 'org.jetbrains.kotlin.jvm' version '1.4.31' apply false
```

Comment:
Hmmm weird that with Kotlin plugin 1.4.x we end up with `kotlin` jars baked into the final zip... let's see if this persist even when IJ 2021.1 is finally released.